README.md

```diff
@@ -1,2 +1,8 @@
-# HW01
+# HW01 Sherzod Tokamov
 Edit the readme
+
+# Student bio
```

Comment:
Hi Sherzod! I am practicing making comments on your code. I like how you made another title here in addition to the "HW01" title.